packages/backend/src/routes/data.js

```diff
@@ -94,7 +94,7 @@ const getRecipeDataForExport = async userId => {
         }
 
         if (location.startsWith('/')) {
-          const data = fs.readFileSync(location);
+          const data = fs.readFileSync(location.replace('/api/images/filesystem', '/rs-media'));
```

Comment:
The replaced path should come from the environment variable `FILESYSTEM_STORAGE_PATH`.

https://github.com/julianpoy/RecipeSage-selfhost/blob/main/docker-compose.yml#L31

Reply:
Done :+1: